HoloRepositoryDemoApplication/Assets/HoloStorageConnector/Prefabs/PrefabsScripts/HologramLoader.cs

```diff
@@ -0,0 +1,32 @@
+﻿using UnityEngine;
+using HoloStorageConnector;
+
+public class HologramLoader : MonoBehaviour
+{
+    public string HologramID = string.Empty;
+    public string SceneName;
+    public string ModelName = "LoadedModel";
+    public Vector3 ModelPosition = new Vector3(0f, 0f, 0f);
+    public Vector3 ModelRotation = new Vector3(0f, 0f, 0f);
+    public float ModelSize = 1.0f;
```

Comment:
Same as in the InstantiationSettings, the default value might better be something smaller